e2e/pages/adf/dialog/shareDialog.ts

```diff
@@ -111,7 +111,7 @@ export class ShareDialog {
     }
 
     calendarTodayDayIsDisabled() {
-        const today = this.dayPicker.element(by.css('.mat-datetimepicker-calendar-body-today')).getText();
+        const today = this.dayPicker.element(by.css('.mat-datetimepicker-calendar-body-today')).getText().toString();
```

Comment:
I think element(...).getText() returns promise.Promise<string>. toString() is incorrectly used in this case.

Reply:
what would you suggest, in terms of usage here? can you please elaborate.

Reply:
why did you add .toString() ? what was the problem you were trying to solve?